src/ProxyManager/Exception/FileNotWritableException.php

```diff
@@ -22,4 +22,13 @@ public static function fromInvalidMoveOperation(string $fromPath, string $toPath
             $toPath
         ));
     }
+
+    public static function fromNotWritableDirectory(string $directory) : self
```

Comment:
While annoying, please write one test that verifies this method (yes, I also do test these)

Reply:
:heavy_check_mark: Tested in https://github.com/Ocramius/ProxyManager/pull/438/commits/2414974aad1df2391e629cf1628ce38bbd6d62bd